bundles/org.eclipse.passage.lic.equinox/src/org/eclipse/passage/lic/internal/equinox/requirements/BundleVendor.java

```diff
@@ -33,7 +34,8 @@ protected BundleVendor(Bundle bundle) {
 
 	@Override
 	public String get() {
-		return bundle.getHeaders().get(Constants.BUNDLE_VENDOR);
+		return Optional.ofNullable(bundle.getHeaders().get(Constants.BUNDLE_VENDOR))//
+				.orElse("Unknown vendor"); //$NON-NLS-1$
```

Comment:
This should be visible for user and i18n is required

Reply:
Done